.shippable.yml

```diff
@@ -55,6 +62,8 @@ build:
             fi;
             echo "- Verify commit message and coding style";
             ./scripts/ci/check-compliance.py || true;
+          elif [ "$IS_PULL_REQUEST" = "true" ]; then
+            export ARCH="-a arm -a arc -a nios2 -a riscv32";
```

Comment:
I really don't like having a hard-coded set of arches here.
Every time we add a new arch support to Zephyr, this obscure script will have to be updated, easy to miss. If we forget, we just silently don't build for that arch.
Is there any better way to do this?